catalog.html

```diff
@@ -0,0 +1,183 @@
+                  <img src="img/product-4.jpg" alt="Глина для укладки волос «American Crew»">
+                </a>
+              </p>
+              <p class="catalog-item-price">
+                <b>500 ₽</b>
+                <a class="button" href="#">Купить</a>
+              </p>
+            </li>
+
+            <li class="catalog-item">
+              <h3>
+                <a href="catalog-item.html">
+                  <span class="catalog-category">Гель для волос</span>
+                  <span class="catalog-item-title">«American Crew»</span>
+                </a>
+              </h3>
+              <p class="catalog-item-image">
+                <a href="catalog-item.html">
+                  <img src="img/product-5.jpg" alt="Гель для волос «American Crew»">
+                </a>
+              </p>
+              <p class="catalog-item-price">
+                <b>300 ₽</b>
+                <a class="button" href="#">Купить</a>
+              </p>
+            </li>
+
+            <li class="catalog-item">
+              <h3>
+                <a href="catalog-item.html">
+                  <span class="catalog-category">Набор для бритьяс</span>
+                  <span class="catalog-item-title">«BAXTER OF CALIFORNIA»</span>
+                </a>
+              </h3>
+              <p class="catalog-item-image">
+                <a href="catalog-item.html">
+                  <img src="img/product-6.jpg" alt="Набор для бритья «BAXTER OF CALIFORNIA»">
+                </a>
+              </p>
+              <p class="catalog-item-price">
+                <b>3900 ₽</b>
+                <a class="button" href="#">Купить</a>
+              </p>
+            </li>
+          </ul>
+
+          <ul class="pagination-list">
+            <li class="pagination-item"><a href="#">1</a></li>
+            <li class="pagination-item pagination-item-current"><a>2</a></li>
+            <li class="pagination-item"><a href="#">3</a></li>
+            <li class="pagination-item"><a href="#">4</a></li>
+          </ul>
+    </section>
+  </main>
+  <footer>
+    <p class="address">
+      БАРБЕРШОП «БОРОДИНСКИЙ» <br>
+      АДРЕС: Г. САНКТ-ПЕТЕРБУРГ, УЛ. Б. КОНЮШЕННАЯ, Д. 19/8 <br>
+      <a href="map.html">КАК НАС НАЙТИ?</a> <br>
+      ТЕЛЕФОН: +7 (495) 666-02-66 <br>
+
+      <div class="footer-social">
+        <b>Давайте дружить!</b>
+        <ul>
+          <li><a class="social-button" href="#">Вконтакте</a></li>
+          <li><a class="social-button" href="#">Фейсбук</a></li>
+          <li><a class="social-button" href="#">Инстаграм</a></li>
+        </ul>
+      </div>
+      <p class="footer-copyright">
+        <b>Разработано:</b> <a class="button" href="https://htmlacademy.ru">HTML Academy</a>
+    </p>
+  </footer>
+
+    <section class="modal modal-login">
+      <h2>Личный кабинет</h2>
+      <p>Введите свой логин и пароль.</p>
+      <!-- Здесь будет форма -->
+      <span class="modal-close">x</span>
+    </section>
+</body>
```

Comment:
```suggestion
</body>
</html>
```
Потерялся закрывающий тег